blocks/library/button/index.js

```diff
@@ -156,6 +156,7 @@ class ButtonBlock extends Component {
 			</span>,
 			focus && (
 				<form
+					key="form-link"
```

Comment:
Was this change intentional?

Reply:
Oops, this was a bug fix I wanted to publish separately and accidentally made it to this PR

Reply:
I can extract it to a separate PR but maybe not worth it